projects/packages/videopress/src/client/admin/components/video-grid/index.tsx

```diff
@@ -32,6 +32,7 @@ const VideoGrid = ( { videos, count = 6, onVideoDetailsClick, loading }: VideoGr
 							<VideoCard
 								id={ video?.id }
 								title={ video.title }
+								privacySetting={ video.privacySetting }
```

Comment:
Although we pass some properties to the `<Video Card />` component, we should not. The only required in terms of getting data is the video ID. The other ones are provided by the `useVideo()` hook, called adequately by the component.

[This is](https://github.com/Automattic/jetpack/blob/f45e61deec79634bafeb8c78b6413be6fb1b5ec4/projects/packages/videopress/src/client/admin/components/video-card/index.tsx#L181) the lin where it calls the hook. It provides some state props, but even is possible to get all video data:

```es6
const { data: video } = useVideo( <video-id> );
```

You can take a look at the [VideoQuickActions component](https://github.com/Automattic/jetpack/blob/f45e61deec79634bafeb8c78b6413be6fb1b5ec4/projects/packages/videopress/src/client/admin/components/video-quick-actions/index.tsx#L241) to get the idea.

I suggest not passing the `privacySetting` prop down and instead picking it up straight from [the ConnectVideoCard component](https://github.com/Automattic/jetpack/blob/f45e61deec79634bafeb8c78b6413be6fb1b5ec4/projects/packages/videopress/src/client/admin/components/video-card/index.tsx#L180-L197).

Reply:
@retrofox Related to this, we are not using the `ConnectVideoCard` component (it was declared but never used). Is it OK to use the cutom hook inside the `VideoCard` itself, or it violates our conventions?

Reply:
> ...we are not using the `ConnectVideoCard` component 

I think we are. ConnectVideoCard is the [default function exported by the file](https://github.com/Automattic/jetpack/blob/3f822142dd9c1a692674a41925dd3c62a921d989/projects/packages/videopress/src/client/admin/components/video-card/index.tsx#L199)

:-)